<?xml version="1.0"?>
 <!-- Based on iliad_single_truck.launch
 I have removed orunav stack and replaced it with standard ros one -->
 
<launch>




    <group ns="robot1">
        
        <node name="joy_node" pkg="joy" type="joy_node">
        </node>
        <node name="tele_joy_node" pkg="teleop_twist_joy" type="teleop_node">
        </node>

        <!-- Parse cmd vel -->
        <node name="vel_caster" pkg="zombie" type="parseVels.py" output="screen">
         <param name="twist_cmd_topic"     value="/robot1/cmd_vel"    />        <!-- In -->
         <param name="ackermann_cmd_topic" value="/robot1/controller/cmd_vel"/>                <!-- out -->
         <param name="distBtBackWheels"           value="0.5"          />         <!-- distance between forks -->
         <param name="backWheelsRadius"           value="0.05"          />        <!-- wheel diameter -->
         <param name="wheelsAxesDist"           value="1.2"          />           <!-- distance between front and rear wheels -->
        </node>
        
    </group>
    
</launch>
 
